src/main/java/org/sagebionetworks/bridge/config/PropertiesConfig.java

```diff
@@ -276,7 +283,14 @@ private Properties collapse(final Properties properties, final String envName) {
                 value = envReader.read(key);
             }
             if (value != null) {
-                collapsed.setProperty(key, value);
+                if (key.startsWith(envName + ".")) {
+                    String strippedName = key.substring(envName.length() + 1);
+                    LOG.debug("Overwrite with Env Key=" + strippedName + " with value=" + value);
+                    collapsed.setProperty(strippedName, value);
```

Comment:
Given that we store AWS keys in env vars, will this accidentally leak our keys into the logs?

Reply:
Good point.  will remove the logging.